Description: do not use bundler in tests
 This will force jbuilder.gemspec to be loaded, which will try to call git to
 get the list of files in the gem; we don't want that.
 .
 This patch is Debian-specific
Author: Antonio Terceiro <terceiro@debian.org>
Forwarded: not-needed

--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,5 +1,3 @@
-require "bundler/setup"
-
 require "active_support"
 require "active_support/core_ext/array/access"
 require "active_support/cache/memory_store"
